common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -660,6 +833,194 @@ def test_can_show_answer(self):
+        """
+        self.answer_problem(correctness)
+        self.problem_page.click_submit()
+        self.assertEqual(self.problem_page.problem_progress_graded_value, score)
+        self.problem_page.click_reset()
+        self.assertEqual(self.problem_page.problem_progress_graded_value, '0/1 point (ungraded)')
+
+    @ddt.data(['correct', 'incorrect'], ['incorrect', 'correct'])
+    @ddt.unpack
+    def test_reset_correctness_after_changing_answer(self, initial_correctness, other_correctness):
+        """
+        Scenario: I can reset the correctness of a multiple choice problem after changing my answer
+
+        Given I am viewing a multiple choice problem
+        When I answer a multiple choice problem InitialCorrectness
+        Then my multiple choice answer is marked InitialCorrectness
+        And I reset the problem
+        Then my multiple choice answer is NOT marked InitialCorrectness
+        And my multiple choice answer is NOT marked OtherCorrectness
+        """
+        self.wait_for_status("unanswered")
+        self.assertTrue(self.is_status_true("unanswered"))
+        self.answer_problem(initial_correctness)
+        self.problem_page.click_submit()
+
+        self.wait_for_status(initial_correctness)
+        self.assertTrue(self.is_status_true(initial_correctness))
+        self.problem_page.click_reset()
+
+        self.assertFalse(self.is_status_true(initial_correctness))
+        self.assertFalse(self.is_status_true(other_correctness))
+
+
+@ddt.ddt
+class MultipleChoiceProblemTypeTestNonRandomized(MultipleChoiceProblemTypeBase, NonRandomizedProblemTypeTestMixin):
+    """
+    Tests for non-randomized multiple choice problem
+    """
+    def get_problem(self):
+        """
+        Creates a {problem_type} problem
+        """
+        # Generate the problem XML using capa.tests.response_xml_factory
+        return XBlockFixtureDesc(
+            'problem',
+            self.problem_name,
+            data=self.factory.build_xml(**self.factory_kwargs),
+            metadata={'rerandomize': 'never', 'show_reset_button': True, 'max_attempts': 3}
+        )
+
+    def test_non_randomized_multiple_choice_with_multiple_attempts(self):
+        """
+        Scenario: I can answer a problem with multiple attempts correctly and still reset the problem
+
+        Given I am viewing a "multiple choice" problem with "3" attempts
+        Then I should see "You have used 0 of 3 attempts" somewhere in the page
+        When I answer a "multiple choice" problem "correctly"
+        Then The "Reset" button does appear
+
+        """
+        self.assertEqual(
+            self.problem_page.submission_feedback,
+            "You have used 0 of 3 attempts",
+            "All 3 attempts are not available"
+        )
+
+        self.answer_problem("correct")
+        self.problem_page.click_submit()
+        self.assertFalse(self.problem_page.is_reset_button_present())
```

Comment:
according to test logic the reset button should still be present unless all 3 attempts are completed